torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -262,6 +262,16 @@ std::vector<xla::ComputationClient::DataPtr> ShardingUtil::OutputHandler(
             shards, GetVirtualDevice().toString(), sharding->shape.value(),
             sharding->sharding));
       }
+    } else if (xla::sys_util::GetEnvBool("XLA_USE_SPMD", false)) {
```

Comment:
If we are going to relie on a env var, make it a staic variable and store it in a helper function. Search for `XLA_USE_BF16` for example. We should not keep quering for the env var.